interface/procedure_tools/labcorp/ereq_form.php

```diff
@@ -749,12 +749,16 @@ function ereqForm($pid, $encounter, $form_id, $reqStr = null, $doDoc = true)
         $DOCUMENT_CATEGORY = $category['id'];
 
         $d = new Document();
-        $docid = $d->createDocument($pid, $DOCUMENT_CATEGORY, $filename, "application/pdf", $mpdfData);
+        $good = $d->createDocument($pid, $DOCUMENT_CATEGORY, $filename, "application/pdf", $mpdfData);
+        if (!empty($good)) {
+            echo $good;
+            exit;
+        }
```

Comment:
this block is throwing me off. so if the document creation works and sends back a docid, it then outputs the docid and then exits?

Reply:
disregard. i get it now :)
(it does not return a document id, duh :) )